packages/manager/modules/sign-up/src/components/country-flag/country-flag.controller.js

```diff
@@ -0,0 +1,19 @@
+export default class CountryFlagCtrl {
+  getCountry() {
+    const country = this.country && this.country.toLowerCase();
+    switch (country) {
```

Comment:
Could be in a constants file:

```javascript
export const COUNTRY_FLAG : {
  we : 'us',
  qc : 'ca',
  asia : 'sg',
  ac : 'gb',
  ta : 'gb',
};
```

and replace the `switch` by `get(COUNTRY_FLAG, country, country)`